Chess-Challenge/src/My Bot/MyBot.cs

```diff
@@ -135,15 +139,15 @@ static bool MovedPeaceMayBeEaten(Move move, Board board)
         }, board, move);
     }
 
-    static Move[] FilterMoves(IReadOnlyList<Move> moves, Board board)
+    static Move[] FilterMoves(IReadOnlyList<Move> moves, Board board, int strategy)
```

Comment:
I would like to rename the parameter in order to make things clear. Like "opponentStrategyToAvoid"

Reply:
done! renamed to `enemyStrategyFilter` :)